source/chapters/djing_with_mixxx.rst

```diff
@@ -303,6 +303,72 @@ and click the :guilabel:`Play` button next to the waveform.
 To display the Preview deck, press :kbd:`CTRL` + :kbd:`4` (Windows/Linux) or
 :kbd:`CMD` + :kbd:`4` (Mac).
 
+.. _djing-gain-staging:
+
+Setting your levels properly (gain staging)
+===========================================
+.. sectionauthor::
+   Be <be.0@gmx.com>
+   
+.. figure:: ../_static/level-meter-green.png
+   :align: left
+   :alt: A level meter in Mixxx with the gain set properly for the loudest part of a track
+   :figclass: pretty-figures
+
+Setting your levels properly, also known as gain staging, is essential for
+getting the best sound quality out of the equipment you are using. At every link
+in your signal chain, from Mixxx's channel gains to the power amplifier, the
+level should be well above the noise floor, but lower than the maximum level
+before the signal clips. The :term:`level meters <level meter>` should mostly be
+around the top of their green region. The level meter pictured to the left shows
+where Mixxx's level meters should average at the loudest parts of tracks. The
+average level should not be in the yellow region. Use the yellow region to leave
+headroom, or available level above the average before the signal clips. The
+loudest parts of the music (the transients) should briefly go into the yellow
+region.
+
+.. figure:: ../_static/level-meter-red.png
+   :align: right
+   :alt: A level meter in Mixxx indicating clipping. The gain should be turned down!
+   :figclass: pretty-figures
+
+**If a level meter is in its red region, the signal is clipping and the gain
+should be turned down.** Some equipment doesn't have a level meter and only has
+an LED that turns on when the signal clips. Clipping means that the peaks of the
+waveform are flattened because the equipment has reached the maximum level that
+it can amplify the signal to. This distorts sound in an unpleasant way and can
+damage equipment by driving it with more power than it is designed to handle.
+Increasing the gain past the point where clipping begins (further into the red
+on a meter) will distort the signal more. If there is a limiter being used with
+the sound system (which is usually the case on large PA systems), clipping more
+will not make the sound any louder but will distort it more. If you want to make
```

Comment:
Ok, to much details. 

Can we just remove the following, since it is unrelated to Mixxx: 

"If there is a limiter being used with the sound system (which is usually the case on large PA systems), clipping more
will not make the sound any louder but will distort it more."